api/config.yaml

```diff
@@ -0,0 +1,170 @@
+openapi: 3.0.0
+components:
+  schemas:
+    Config:
+      type: object
+      properties:
+        telemetry:
+          $ref: "./components/schemas/TelemetryConfig"
+        server:
+          $ref: "./components/schemas/Server"
```

Comment:
This PR is only for data stores, right? This config won't allow users to set polling timeouts and retry delays.

Reply:
Exactly, not yet, I just wanted to focus on data stores & exporters